source/renderer/app/components/settings/categories/StakePoolsSettings.js

```diff
@@ -313,7 +318,7 @@ export default class StakePoolsSettings extends Component<Props, State> {
             successfullyUpdated={false}
           />
         )}
-        {smashServerType === SMASH_SERVER_TYPES.IOHK && (
+        {!isSyncing && smashServerType === SMASH_SERVER_TYPES.IOHK && (
           <div className={styles.optionDescription}>
```

Comment:
@daniloprates please extract these repetitive parts into "renderOptionDescription()" method or something similar so that you can on top of it simply handle syncing state without needing to repeat `!isSyncing` on every  `smashServerType` check 🙏

Reply:
✅